rotkehlchen/db/settings.py

```diff
@@ -61,6 +61,7 @@
 DEFAULT_AUTO_DELETE_CALENDAR_ENTRIES = True
 DEFAULT_AUTO_CREATE_CALENDAR_REMINDERS = True
 DEFAULT_ASK_USER_UPON_SIZE_DISCREPANCY = True
+DEFAULT_AUTO_DETECT_TOKENS = True
```

Comment:
This should be false.

Reply:
it should be efficient enough now so keeping the default to true (https://discord.com/channels/657906918408585217/1080783409640648765/1248392708624613429)